web/src/emoji_picker.ts

```diff
@@ -622,24 +700,24 @@ function get_default_emoji_popover_options() {
                 },
             ],
         },
-        onCreate(instance) {
+        onCreate(instance: PopoverInstance) {
             emoji_popover_instance = instance;
             const $popover = $(instance.popper);
             $popover.addClass("emoji-popover-root");
             instance.setContent(
-                ui_util.parse_html(generate_emoji_picker_content(current_message_id)),
+                ui_util.parse_html(generate_emoji_picker_content(current_message_id ?? 0)),
```

Comment:
I can not understand the logic behind defaulting this to 0 .Would be helpful if you could clarify this please.

Reply:
Yeah, I should have asserted instead of coalescing, thanks for pointing this out!